docs/workflow/testing/libraries/testing-android.md

```diff
@@ -154,3 +154,29 @@ The emulator can be launched with a variety of options. Run `emulator -help` to
 ### Debugging the native runtime code using Android Studio
 
 See [Debugging Android](../../debugging/mono/android-debugging.md)
+
+## Upgrading the Android NDK Version in CI Pipelines
+
+The Android NDK LTS version is released annually. CI pipelines retrieve the NDK version from Docker images hosted in the [dotnet-buildtools-prereqs-docker](https://github.com/dotnet/dotnet-buildtools-prereqs-docker) repository.
+For reference, see an example Dockerfile:
+[Azure Linux 3.0 .NET 10.0 Android Dockerfile](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/azurelinux/3.0/net10.0/android/amd64/Dockerfile).
+
+To upgrade the NDK version used in CI for building and testing Android, follow these steps:
+
+### 1. Verify the New NDK Version Locally
+- Download the new NDK version.
+- Test the local build using the new NDK by building a sample Android app.
+- Ensure **AOT** and **AOT_WITH_LIBRARY_FILES** are enabled in the build.
+
+### 2. Test the New NDK in CI and Fix Issues
+- Create a new Docker image containing the updated NDK version.
```

Comment:
This seems to be missing some important steps. How does that related to the prereqs repo?